src/plugins/child.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import path from 'path';
+import config from '../config';
+
+import { notificationBus, EVENTBUS_EVENT_NAME } from '../services/notification.service';
+
+/**
+ * @typedef {Object} Plugin
+ * @property {Function} init
+ * @property {Function} [destroy]
+ */
+
+/**
+ * @typedef {Object} PluginConfig
+ * @property {string} name Plugin name
+ * @property {string} file Plugin filename inside the plugin directory.
+ */
+
+/**
+ * Hathor plugins
+ * @type {Record<string, PluginConfig>}
+ */
+export const hathorPlugins = {
+  debug: {
+    name: 'debug',
+    file: 'hathor_debug.js',
+  },
+  ws: {
+    name: 'websocket',
+    file: 'hathor_websocket.js',
+  },
+  sqs: {
+    name: 'sqs',
+    file: 'hathor_sqs.js',
+  },
+  rabbitmq: {
+    name: 'rabbitmq',
+    file: 'hathor_rabbitmq.js',
+  },
+};
+
+/**
+ * Get the module path from the plugin filename.
+ *
+ * @param {string} pluginFile Plugin file name
+ *
+ * @returns {string} Plugin module path
+ */
+export const getPluginPath = pluginFile => {
+  const pluginDir = config.pluginDir || './src/plugins';
```

Comment:
I didn't see this `pluginDir` in any docs. Why do we need it?

Reply:
We don't, it was an earlier concept of having a dev defined plugin directory, but we have changed it so i removed it from the configs.